wgsl/index.bs

```diff
@@ -3412,6 +3412,39 @@ TODO: *Stub*. Call to function that has a [=return type=] is an expression.
 
 TODO: *Stub*: how to write each of the abstract pointer operations
 
+## Constant Identifier Expression  ## {#constant-identifier-expr}
+
+<table class='data'>
+  <caption>Getting the value of a `let`-declared identifier</caption>
+  <thead>
+    <tr><th>Precondition<th>Conclusion<th>Description
+  </thead>
+  <tr algorithm="pipeline-overridable constant value">
+       <td>
+          |c| is an identifier [=resolves|resolving=] to
+          an [=in scope|in-scope=] [=pipeline-overridable=] `let` declaration with type |T|
+       <td class="nowrap">
+          |c| : |T|
+       <td>If pipeline creation specified a value for the constant ID, then the result is
```

Comment:
It's worth spelling out here that this results in different entry points having a different value for this constant. This isn't obvious given the current stress on "module-scope".

Reply:
Changed this to:

       <td>If pipeline creation specified a value for the [=pipeline constant ID|constant ID=],
           then the result is that value.
           This value may be different for different pipeline instances.<br>
           Otherwise, the result is the value computed for the initializer expression.
           Pipeline-overridable constants appear at module-scope, so evaluation occurs
           before the shader begins execution.